### DISCUSSION TITLE
Add ability to view a score on web from the results screen

### DISCUSSION THREAD

**Discussion by Game4all** [ppy/osu#13529] [Ideas]
osu!web now has a quite nice page to view score details of a player on a beatmap along with a button to download the replay which offers an alternative to screenshots to share a score and its replay on messaging apps with just an URL. A button could be added to the results screen next to the other buttons to allow opening a link to the score in the system browser to allow quick sharing of a score (as seen on the screenshot here)

![osu_2021-06-16_17-40-46](https://user-images.githubusercontent.com/20256717/122250526-0656ce80-ceca-11eb-9a14-d9805a4ebea7.jpg)

This could be of course further integrated in chat inside lazer to display links pointing to scores in a way similar to how it is already done with beatmaps.

**Comment by peppy** [comment]
If this is to be implemented it should be done so like on other overlays / screens (an inline link somewhere on the score display, not its own button). Note that currently this would only work for osu-stable scores, too.

**Reply by Game4all** [reply]
Also noticed that scores in multiplayer playlists get their online id populated but the matching web route to display them doesn't exist (yet).

So yeah you're right , it may be too soon to consider this feature.
